templates/node-class.mustache

```diff
@@ -14,6 +14,15 @@ var {{&className}} = (function(){
         this.domain = domain;
     }
 
+    {{#isSecure}}
```

Comment:
@borisirota maybe these constructor now need to take an object of options as a parameter?
```javascript
new ProtectedAPI({
    domain: 'http://www.example.org/api',
    token: {
      value: ...
      ...
   }
});
```